changelog.d/5925.removal

```diff
@@ -0,0 +1 @@
+Remove admin/v1/users_paginate endpoint. Contributed by Awesome Technologies Innovationslabor GmbH.
```

Comment:
... and add pagination to `/users`

Reply:
There is `changelog.d/5925.feature`. Also the v1/users endpoint does not support pagination.